openshift/templates/backups/README.md

```diff
@@ -56,4 +56,43 @@ gzip -cd ~/backups/db/daily/test.sql.gz | mysql -u $WORDPRESS_DB_USER -p$(cat $M
 * Use the sidecar, see instructions for deploying the sidecar.
 * Do a ```rsync -a --exclude=restore /home/sidecar/backups/html/ /var/www/html/``` 
     * you might have to delete the files in /var/www/html, but **WARNING** make sure you know what you are doing.
-* Switch from backup volume to regular volume as indicated below, so now WordPress should be using restored files.
\ No newline at end of file
+* Switch from backup volume to regular volume as indicated below, so now WordPress should be using restored files.
+
+### Manual backups/site duplication
+At times you may wish to back up the database manually - for example, when copying a site or in preparation for a potentially disruptive operation.
+1. Export the database by running the following command from the `sidecar of the original site`. Replace example.gov.bc.ca with the URL of the site.
+    ```bash
+    wp db export --tables=$(wp db tables --url=old-example.url/ --path=/var/www/html/ --all-tables-with-prefix --format=csv) --url=old-example.url/ --path=/var/www/html/ backup.sql
+    ```
+2. If the site to be backed up is part of a multisite deployment and you are copying/restoring it to another multisite deployment, replace all instances of the old site id with the new one (if it is different). To find the id of the site, navigate to the `Sites` dashboard page and hover over the site. the URL will end in `?id=[site-id]`.
+   * Note: If the new deployment is not multisite, new-site-id will be an `empty string`.
+    ```bash
+    vi backup.sql :%s/wp_[example-site-id]_/wp_[new-site-id]/g
+    ```
```

Comment:
it might be clearer to say:

`vi backup.sql`
and once you are in the editor, enter the following in command mode: 

`:%s/wp_[example-site-id]_/wp_[new-site-id]/g`

Reply:
A sensible suggestion. Thanks @robrien-bcgov.